src/index.js

```diff
@@ -782,19 +785,19 @@ const members = {
           return this.__removeChild(refNode);
         }
 
-        return removeNodeFromNode(this, refNode);
+        return removeNodeFromNode(this, refNode) || refNode;
```

Comment:
Maybe making these into `else if` blocks and then having a single `return refNode` at the bottom would remove some of the duplication?